template/requirements/docs.in

```diff
@@ -11,3 +11,5 @@ sphinx-design
 # Temporary until questionary (dep of copier) updates
 # See https://github.com/tmbo/questionary/blob/2df265534f3eb77aafcf70902e53e80beb1793e0/pyproject.toml#L36C43-L36C110
 prompt-toolkit==3.0.36
+# Temporary until prompt-tookit conflict is resolved.
+ipython==8.9.0
```

Comment:
This didn't work for me in `esssans` this morning, can you try if you get it to work there, from this branch?

Reply:
Yeah I just tried to. It should be in `base.in` instead... because of ``pythreejs``.

They both needed to be pinned in the ``base.in``.
Does it make sense to put those temporary parts always in ``base.in``...?